src/Tests/dotnet-new.Tests/TemplateEngineSamplesTest.cs

```diff
@@ -52,6 +53,7 @@ public TemplateEngineSamplesTest(SharedHomeDirectory sharedHome, ITestOutputHelp
             _log.LogInformation($"Template with {caseDescription}");
             Dictionary<string, string> environmentUnderTest = new() { ["DOTNET_NOLOGO"] = false.ToString() };
             TestContext.Current.AddTestEnvironmentVariables(environmentUnderTest);
+            FileExtensions.AddTextExtension(".cshtml");
```

Comment:
I believe you need to specify the extensions without the dot in this API

Reply:
Though the build succeeds now, which is quite strange

Reply:
Oh - it actualy strips the dot (https://github.com/VerifyTests/EmptyFiles/blob/79678f2f459d7969425fbdbb0cad3cc88677d728/src/EmptyFiles/FileExtensions.cs#L17), unlinke some other Verify APIs.

So all is good